docs/reference/quickstart/aggs-tutorial.asciidoc

```diff
@@ -0,0 +1,530 @@
+     "date_histogram": {
+       "field": "order_date",
+       "calendar_interval": "day",
+       "format": "yyyy-MM-dd"
+     },
+     "aggs": {
+       "revenue": { <1>
+         "sum": {
+           "field": "taxful_total_price"
+         }
+       },
+       "unique_customers": { <2>
+         "cardinality": {
+           "field": "customer_id"
+         }
+       },
+       "avg_basket_size": { <3>
+         "avg": {
+           "field": "total_quantity"
+         }
+       }
+     }
+   }
+ }
+}
+----
+// TEST[skip:Using Kibana sample data]
+<1> Daily revenue
+<2> Number of unique customers each day
+<3> Average number of items per order
+
+[discrete]
+[[aggregations-tutorial-trends]]
+=== Track trends and patterns
+
+You can use <<search-aggregations-pipeline,pipeline aggregations>> on the results of other aggregations.
+Let's analyze how metrics change over time.
+
+[discrete]
+[[aggregations-tutorial-moving-average]]
+==== Smooth out daily fluctuations
+
+Moving averages help identify trends by reducing day-to-day noise in the data.
+Let's observe sales trends more clearly by smoothing daily revenue variations.
+
+[source,console]
+----
+GET kibana_sample_data_ecommerce/_search
+{
+  "size": 0,
+  "aggs": {
+    "daily_sales": {
+      "date_histogram": {
+        "field": "order_date",
+        "calendar_interval": "day"
+      },
+      "aggs": {
+        "daily_revenue": {  <1>
+          "sum": {
+            "field": "taxful_total_price"
+          }
+        },
+        "smoothed_revenue": { <2>
+          "moving_fn": { <3>
+            "buckets_path": "daily_revenue", <4>
+            "window": 3, <5>
+            "script": "MovingFunctions.unweightedAvg(values)" <6>
+          }
+        }
+      }
+    }
+  }
+}
+----
+// TEST[skip:Using Kibana sample data]
+<1> Calculate daily revenue first
+<2> Create a smoothed version of the daily revenue
+<3> Use `moving_fn` for moving window calculations
+<4> Reference the revenue from our date histogram
+<5> Use a 3-day window — use different window sizes to see trends at different time scales
+<6> Use the built-in unweighted average function
+
+.Example response (truncated)
+[%collapsible]
+====
+[source,console-result]
+----
+{ ...
+  "aggregations": {
+    "daily_sales": {
+      "buckets": [
+        {
+          "key_as_string": "2024-10-24T00:00:00.000Z", <1>
+          "key": 1729728000000,
+          "doc_count": 146, <2>
+          "daily_revenue": {
+            "value": 10578.53125 <3>
+          },
+          "smoothed_revenue": {
+            "value": null <4>
+          }
+        },
+        {
+          "key_as_string": "2024-10-25T00:00:00.000Z",
+          "key": 1729814400000,
+          "doc_count": 153,
+          "daily_revenue": {
+            "value": 10448
+          },
+          "smoothed_revenue": {
+            "value": 10578.53125 <5>
+          }
+        },
+        ...
+----
+// TEST[skip:Using Kibana sample data]
+<1> Date of the bucket in ISO format
```

Comment:
Can we call out that we got the date in this format because its the default and we didn't specify a different format?